### ISSUE TITLE
Please fix Security advisory for `Cryptography` 5.0

### ISSUE THREAD

**Issue by Smurf-IV** [zeromq/netmq#1068]
As seen in the project file:

![image](https://github.com/zeromq/netmq/assets/2418812/b4f972e6-fd97-4bd3-b9d8-adec7b3f2832)

**Comment by james1301** [comment]
Yes any update on this? This is being shown constantly now by Visual Studio as it now shows transitive packages. Please update this to a safe version.

**Comment by KL-Sven** [comment]
+1

**Comment by tmatthey** [comment]
Net7 or older have already reached end of support and I would not expect any updates. From my findings most packages are bumped to most current available versions, at least I could
not see any big improvements with a local upgrade of packages. Either netmq has to stick to Net8 or Net9 only, or you override the outdated packages with vulnerabilities in your application. That said, this package shall not override any transient packages, only the top most app using netmq, which will define the min runtime and can depend on packages with older .net, but not the other way. 

https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core
